devtools/debug_local.py

```diff
@@ -48,6 +48,7 @@
     "KQLQueryset",
     "Reflex",
     "Eventstream",
+    "MLExperiment",
```

Comment:
Revert all changes to original in this file, it is not necessary to include the MLExperiment item type in scope in the debug file.